recipes/python-keycloak/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "python-keycloak" %}
+{% set version = "0.18.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: c1ffcdfd8f770774c02b7cbfacdba9d7ef94c0be4e42fabe49308e436ba21490
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - python-jose >=1.4.0
```

Comment:
Can you double check if python-jose and requests are needed during the installation phase?